src/main/kotlin/org/rust/lang/core/types/Extensions.kt

```diff
@@ -80,5 +80,16 @@ fun Ty.builtinDeref(explicit: Boolean = true): Pair<Ty, Mutability>? =
 
 val RsExpr.isDereference: Boolean get() = this is RsUnaryExpr && this.mul != null
 
-
 val RsExpr.isMutable: Boolean get() = mutabilityCategory?.isMutable ?: Mutability.DEFAULT_MUTABILITY.isMut
+
+tailrec fun unwrapParenExprs(expr: RsExpr): RsExpr =
+    when (expr) {
```

Comment:
I don't like `if expr`s myself, but there are only two options
We should decide something for such cases